internal/namespaces/config/commands.go

```diff
@@ -749,3 +787,67 @@ func getProfile(config *scw.Config, profileName string) (*scw.Profile, error) {
 	}
 	return profile, nil
 }
+
+func validateProfile(profile *scw.Profile) error {
+	if profile.AccessKey != nil {
+		if *profile.AccessKey == "" {
+			return &core.CliError{
+				Err: fmt.Errorf("access key cannot be empty"),
+			}
+		} else if !validation.IsAccessKey(*profile.AccessKey) {
+			return core.InvalidAccessKeyError(*profile.AccessKey)
+		}
```

Comment:
```suggestion
		if *profile.AccessKey == "" {
			return &core.CliError{
				Err: fmt.Errorf("access key cannot be empty"),
			}
		}
		
		if !validation.IsAccessKey(*profile.AccessKey) {
			return core.InvalidAccessKeyError(*profile.AccessKey)
		}
```
No need for `else` (same for other checks)